strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/storage/repository/Repository.java

```diff
@@ -57,6 +57,9 @@
     @XmlAttribute
     private String status = RepositoryStatusEnum.IN_SERVICE.getStatus();
 
+    @XmlAttribute
+    private Long artifactsMaxSize;
```

Comment:
1. Could you please rename the variable to `artifactMaxSize`?
2: Coule you please set the `@XmlAttribute(name = "artifact-max-size`, so that the attribute matches the convention of the rest of the XML attributes in the project?